README.md

```diff
@@ -1 +1,53 @@
-src for the 1.16.4 chipped mod o/
+# Chipped
+
+## Description
+
+Have you ever wanted to build an epic sandstone castle? Or maybe a nether fortress made out of pure magma? Perhaps you are just here to build a house out of clay or a totempole with the infamous diorite faces? Either way, we got you covered!
+
+Chipped is a 1.16 building mod that expands on the variety of blocks you can use in-game. The mod offers overs 2000 unique building blocks, from stained glass variations all the way to concrete, prismarine and even gilded blackstone! Not only are all the textures hand-painted in the new vanilla style, but they also include some connected textures! Inspired by the popular Chisel mod, Chipped allows builders to have their full range of creativity, because every block deserves a friend!
+
+Chipped will aim to support all Minecraft versions 1.16+. We have only recently released, so making variations for all vanilla textures is our current priority. In the future, Chipped is planning to have compatibility with other popular mods.
+
+You are free to use this mod in your modpacks! Please just provide credit (:
+
+## Important Notes
+
+With the release of our newest update 1.05. We added a bunch of connected textures! To have these ctm textures work in-game, please download the mod found on the [Connected Textures Mod Curseforge page](https://www.curseforge.com/minecraft/mc-mods/chipped). We plan to release more cool blocks in future updates so stay tuned!
```

Comment:
shouldn't this link to CTM instead?

Reply:
ill fix the link, it was an error on curse we realized just a bit ago, thank you guys!